include/onnxruntime/core/graph/graph.h

```diff
@@ -1153,7 +1154,7 @@ class Graph {
   void SetInputs(gsl::span<const NodeArg* const> inputs);
 
   void SetInputs(std::initializer_list<const NodeArg*> inputs) {
-    SetInputs(gsl::make_span(inputs));
```

Comment:
MS GSL's make_span() didn't compile with some of these original usages, hence the replacement with AsSpan